### ISSUE TITLE
Home Page - Work Section

### ISSUE THREAD

**Issue by chankdawg** [solomon71/tbx-olympus#67]
![Image](https://github.com/solomon71/tbx-olympus/assets/125610954/6d15a562-917e-4f0f-bbd4-fbabc93aac3b)

Figma Link: https://www.figma.com/file/toPXlIHGxdI2lUbKaMI6tz/Intr?type=design&node-id=3536-685&mode=design&t=ek9A7WbRJOOiAVWr-0

Requirements:
Images and copy can be added and edited in the google doc
Clicking on the image or the type should take you to the case study/work detail page
When you scroll to each item it should fade up and reveal.

- Basically a match to what exists in the current site with the two-column layout and different aspect ratio for the image.
